common/app/views/fragments/items/elements/facia_cards/image.scala.html

```diff
@@ -1,16 +1,28 @@
-@(imageContainer: model.ImageContainer, inlineImage: Boolean = false, maxWidth: Int = 620)(implicit request: RequestHeader)
+@(imageContainer: model.ImageContainer, inlineImage: Boolean = false, maxWidth: Int = 940, widthsByBreakpoint: Option[layout.WidthsByBreakpoint] = None)(implicit request: RequestHeader)
 
-@import views.support.{ImgSrc, Item300}
+@import layout.Source
+@import views.support.ImgSrc
 
-@ImgSrc.imager(imageContainer, maxWidth).map { imgSrc =>
-    <div class="fc-item__media-wrapper">
-        <div class="fc-item__image-container u-responsive-ratio js-image-upgrade @if(inlineImage){inlined-image}"
-             data-src="@imgSrc">
-            @if(inlineImage) {
-                @Item300.bestFor(imageContainer).map{ url =>
-                    <img src="@url" class="responsive-img" alt="" role="presentation" />
+<div class="fc-item__media-wrapper">
+    <picture class="fc-item__image-container u-responsive-ratio @if(inlineImage) {inlined-image}">
+        <!--[if IE 9]><video style="display: none;"><![endif]-->
```

Comment:
Might be worth adding a comment to that effect in the template. You can do so like this

```
@* Your comment *@
```

and nothing will render client side.